tests/test_moe/test_moe_ep_tp.py

```diff
@@ -21,7 +133,14 @@ def run_test(rank: int, world_size: int, port: int, num_experts: int, batch_size
     local_model = SparseMLP(num_experts=num_experts, hidden_size=dim, intermediate_size=dim * 2)
     MOE_MANAGER.__init__()
     MOE_MANAGER.setup(parallel="EP")
-    ep_model = SparseMLP(num_experts=num_experts, hidden_size=dim, intermediate_size=dim * 2)
+    os.environ["LOCAL_WORLD_SIZE"] = str(world_size)
+    enable_hierarchical_comm = torch.__version__ >= "1.13.1"
```

Comment:
You can use processgroupmesh to get group ranks instead of restricting torch version